newsroom/users/views.py

```diff
@@ -156,7 +151,7 @@ async def search(args: None, params: SearchArgs, request: Request) -> Response:
         lookup = {"_id": {"$in": params.ids}}
 
     if params.sort:
-        sort = params.sort
+        sort = str(params.sort)
```

Comment:
Conversion to string not required, it's either None or string already

Reply:
indeed.